src/main/java/org/springframework/cloud/deployer/spi/kubernetes/AbstractKubernetesDeployer.java

```diff
@@ -125,7 +118,7 @@ protected RuntimeEnvironmentInfo createRuntimeEnvironmentInfo(Class spiClass, Cl
 		return map;
 	}
 
-	protected AppStatus buildAppStatus(String id, PodList podList, ServiceList services) {
+	AppStatus buildAppStatus(String id, PodList podList, ServiceList services) {
```

Comment:
This method can be left as protected. Seems like it is only used in child classes.